packages/documentsearch-extension/src/index.ts

```diff
@@ -74,18 +75,21 @@ const labShellWidgetListener: JupyterFrontEndPlugin<void> = {
  * Initialization data for the document-search extension.
  */
 const extension: JupyterFrontEndPlugin<ISearchProviderRegistry> = {
-  id: '@jupyterlab/documentsearch:plugin',
+  id: '@jupyterlab/documentsearch-extension:plugin',
```

Comment:
Could you mention it in the 3 to 4 migration guide please?

Reply:
Done. Thx.